########## Testcase 0: Failed ##########
Expected Output: 
Yes
Your Output: 
No
----------------------------------------
########## Testcase 1: Failed ##########
Expected Output: 
No
Your Output: 
Exception in thread "main" java.lang.NullPointerException
	at RansomNote.checking(Solution.java:224)
	at Solution.main(Solution.java:272)
----------------------------------------
########## Testcase 2: Passed ##########
Expected Output: 
No
Your Output: 
No
----------------------------------------
########## Testcase 3: Failed ##########
Expected Output: 
No
Your Output: 
Exception in thread "main" java.lang.NullPointerException
	at RansomNote.checking(Solution.java:224)
	at Solution.main(Solution.java:272)
----------------------------------------
########## Testcase 4: Failed ##########
Expected Output: 
Yes
Your Output: 
No
----------------------------------------
########## Testcase 5: Failed ##########
Expected Output: 
Yes
Your Output: 
Exception in thread "main" java.lang.NullPointerException
	at RansomNote.checking(Solution.java:224)
	at Solution.main(Solution.java:272)
----------------------------------------
